AppController/lib/haproxy.rb

```diff
@@ -54,6 +54,36 @@ module HAProxy
   ALB_SERVER_TIMEOUT = 300000
 
 
+  # The position in the haproxy profiling information where the name of
+  # the service (e.g., the frontend or backend) is specified.
+  SERVICE_NAME_INDEX = 1
+
+
+  # The position in the haproxy profiling information where the number of
+  # enqueued requests is specified.
+  REQ_IN_QUEUE_INDEX = 2
+
+
+  # The position in the haproxy profiling information where the number of
+  # current sessions is specified.
+  CURRENT_SESSIONS_INDEX = 4
+
+
+  # The position in the haproxy profiling information where the status of
+  # the specific server is specified.
+  SERVER_STATUS_INDEX = 17
+
+
+  # The position in the haproxy profiling information where the total
+  # number of requests seen for a given app is specified.
+  TOTAL_REQUEST_RATE_INDEX = 48
+
+
+  # The String haproxy returns when we try to set a parameter on a
+  # non defined server or backend.
+  NO_SUCH = "No such"
```

Comment:
You clearly saw into my world domination plan! Any suggestion on what name should I use?